src/main/java/io/cdap/plugin/cdc/source/salesforce/records/ChangeEventHeader.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.cdc.source.salesforce.records;
+
+import java.util.Arrays;
+
+/**
+ * Contains information about change event. Should be used instead of {@link com.sforce.soap.partner.ChangeEventHeader}
+ * because GSON does not support setters.
```

Comment:
I don't understand what this means. Why would we want setters? Why does this class have setters?